Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,236 @@
+
+- Get the feature position ``featurePos`` for all pulses which belong to the
+  same set. Store these feature positions using their sweep number and pulse
+  index as key.
+- Now shift all pulses in all sets from the same region by ``-featurePos``
+  where ``featurePos`` is used from the same sweep and pulse index.
+
+.. _db_paplot_operation_order:
+
+Operation order
+^^^^^^^^^^^^^^^
+
+Data operations occur in the following (fixed) order:
+
+- Gather pulses
+- Pulse sorting
+- Failed pulse search
+- Zeroing
+- Time alignment
+- Averaging
+
+PA plot settings
+^^^^^^^^^^^^^^^^
+
+The following sections describe PA settings. Settings are configured on the
+data browser PA settings tab (shown below).
+
+.. _Figure Pulse Average Browser Settings:
+
+.. figure:: BrowserSettingsPanel-PA-plot.png
+   :align: center
+
+   Settings for the pulse average plot
+
+Pulse Averaging
+"""""""""""""""
+
+- ``enable``: Toggle the display of the PA image/trace plots.
+   To adjust multiple settings at-a-time, disable the PA plot, adjust settings, and re-enable.
+
+Image settings
+""""""""""""""
+
+- ``Enable image plot``: Toggle the display of the :ref:`Figure Pulse Average Image plot`
+- ``Draw X zero line``: Draws vertical line at ``X == 0`` in each column. Requires time alignment
+   (see below). Facilitates measurement of event latency.
+- ``Popup Menu``: Color scheme for image plots
+- ``Sort order``: Sort order of the displayed pulses. For ``Sweep`` the pulses are
+  first ordered by sweep number and then by pulse index. For ``Pulse`` it is
+  first pulse index and then sweep number. See also
+  :ref:`db_paplot_image_sortorder`.
+
+Trace settings
+""""""""""""""
+
+- ``Enable trace plot``: Toggle the display of the :ref:`Figure Pulse Average Trace plot`.
+- ``Vertical scale bar``: Size of the vertical scale bar in y-axis units
+
+Deconvolution
+"""""""""""""
+
+- ``Deconvolution``: Enables deconvolution [#1]_ of the average pulse.
+  Deconvolution trace is displayed with the average trace in the trace plot and
+  immediately above the average line in the image plot.
+- ``Tau [ms]``: Time constant [#2]_
+- ``Smoothing``: Smoothing parameter, use ``1`` to disable smoothing
+- ``Display [ms]``: Time range of the average pulse used for the deconvolution, use ``inf`` to use the full range
+
+General
+"""""""
+
+- ``Zero pulse``: Toggle Pulse Zeroing. Zeroing is carried out by
+  differentiation, followed by the integration of each pulse.
+- ``Time alignment``: Toggle time alignment of pulses from one column. See
+  :ref:`db_paplot_timealignment` for an in-depth explanation.
+- ``Show average``: Toggle average pulse display. For the image plot, the
+  average is the bottom-most row. Failed pulses (see below) are not included
+  in the average.
+- ``Use multiple graphs``: Creates a panel for each PA set. Normally, multiple
+  PA sets are distributed onto a single panel.
+  graph is used for all traces and images, when checked a separate graph is
```

Comment:
the sentence seems to start in the middle

i.e.,
graph is used......

Reply:
left over from the old text.